compiler/src/dmd/tokens.d

```diff
@@ -23,6 +23,11 @@ import dmd.common.outbuffer;
 import dmd.root.rmem;
 import dmd.root.utf;
 
+/* if you change this, remember to update:
+ compiler/test/unit/lexer/location_offset.d
+ compiler/src/tests/cxxfrontend.cc
+*/
```

Comment:
I get why you put this here, but what actually matters is updating the corresponding header.
I didn't hit this problem changing it when altering existing members when fixing up `-HC` C++ header generation.

Reply:
I did update location_offset.d. I cannot find anything wrong with cxxfrontend.cc. Tests should be designed to make it easy to find mistakes, not just fail with a generic message and no information whatsoever about what failed and where.

This isn't the first time I've been faced with this awfulness, and it's always a large time sink to find it.

Reply:
Should be : `compiler/src/dmd/token.h`, not `cxxfrontend.cc`.

Reply:
Sometimes cxxfrontend.cc will need updating, too. Added tokens.h.